src/sentry/static/sentry/app/data/platformCategories.tsx

```diff
@@ -160,6 +160,13 @@ export const performance: PlatformKey[] = [
   'node-connect',
 ];
 
+/**
+ * Additional aliases used for filtering in the platform picker
+ */
+export const filterAliases: Partial<Record<PlatformKey, string[]>> = {
+  native: ['cpp', 'c++'],
```

Comment:
does this work if I search for "c+"?

Reply:
Yes it does

Reply:
the filter does substring matching, so just typing a partial part of the alias will result in a match.